src/OrleansCodeGenerator/RoslynCodeGenerator.cs

```diff
@@ -89,8 +89,14 @@ public void GenerateAndLoadForAssemblies(params Assembly[] inputs)
             }
 
             var timer = Stopwatch.StartNew();
+            var emitDebugSymbols = false;
             foreach (var input in inputs)
             {
+                if (!emitDebugSymbols)
+                {
+                    emitDebugSymbols = RuntimeVersion.IsAssemblyDebugBuild(input);
```

Comment:
we actually do need `|=` as it was before, don't we? Otherwise last assembly will define this value for the entire set